R/sacramento.R

```diff
@@ -120,3 +296,6 @@ sacramento.ranges <- function() {
     pfree = c(0, 0.6)
   )
 }
+
+#' @import utils
+utils::globalVariables(c("sma_sac_state"))
```

Comment:
I wouldn't have thought this was needed?

Reply:
Please see similar comments above re: global variables and R CMD check.